app/src/main/java/com/github/h3lp3rs/h3lp/database/models/EmergencyInformation.kt

```diff
@@ -0,0 +1,15 @@
+package com.github.h3lp3rs.h3lp.database.models
+
+import java.util.*
+
+/**
+ * A model describing an emergency information object
+ */
+data class EmergencyInformation(
+    val id: String = UUID.randomUUID().toString(),
+    val latitude: Double,
+    val longitude: Double,
+    val meds: ArrayList<String>,
+    val time: Date,
+    val medicalInfo: String
```

Comment:
Maybe add a comment here about what the medicalInfo should include.

Reply:
We can probably leave it as is for now, or define another class to represent that => Don't we have one already?

Reply:
Yes but we save it as a JSON : String in the database, but originally it's of time MedicalInformation

Reply:
I meant to ask her to just add a comment about what she meant by medicalInfo, because I didn't understand at first what she meant by it. Looking at the changes she made now, she intends to send the entire medical card to the database which should indeed be a separate object and not a String as you said.  However, this is an entirely different task, and as we are dealing with the user's sensitive data, we have to first ask the user to choose which information he is ok with sending to the database and thus share with other users ( blood type, allergies, current medication...). This could simply be a couple of toggle buttons in the settings activity and depending on their value we would choose which elements to send to the database.